src/Datadog.Trace/Tracer.cs

```diff
@@ -20,8 +22,11 @@ public class Tracer : IDatadogTracer
 
         private static readonly Vendors.Serilog.ILogger Log = DatadogLogging.For<Tracer>();
 
+        private static int _tracerCount;
```

Comment:
Just curious if you know how many tracer instances are typically created in a busy app in a day using our .NET Tracer?

Reply:
One: the static `Tracer.Instance`.

Reply:
One: the static `Tracer.Instance`.

Reply:
Right, but the tracer count you're keeping is associated with how many times it is requested? or what?

Reply:
I would also appreciate a little more description on what this count serves

Reply:
Right now it counts the number of `Tracer` instances created, (which will be one in most applications). The count is used as the `value` for the heartbeat metrics. The idea is that it will tell us how many Tracers are alive for a given host/env/service/etc.

You made me realize we are not decrementing the count, so I just added a destructor to decrement this count (21cf382695a1985d46c48e2e78863a9c6e6f21ff).

Reply:
> a little more description on what this count serves

It's use as the value for the heartbeat metric (gauge). I thought it would be a little more useful to know _how many_ tracer are alive rather than just using `1` all the time to mean "there's at least one tracer alive."